src/signer/src/main/java/ee/ria/xroad/signer/tokenmanager/token/SoftwareTokenWorker.java

```diff
@@ -197,6 +195,7 @@ protected void deleteCert(String certId) throws Exception {
         Signature signature = Signature.getInstance(SIGNATURE_ALGORITHM);
```

Comment:
Sorry, my bad. I removed paranoid and invalid data length check (digest length vs _DigestInfo_ ASN.1 structure length in case SHA\*withRSA algorithms). Let's check only signature algorithm there (in case SHA\*withRSAandMGF1 algorithms are unexpectedly used).